pages/linux/amixer.md

```diff
@@ -1,6 +1,7 @@
 # amixer
 
 > Mixer for ALSA soundcard driver.
+> More information: <https://wiki.archlinux.org/title/Advanced_Linux_Sound_Architecture>.
```

Comment:
```suggestion
> More information: <https://manned.org/amixer>.
```
If the command isn't Arch specific, we generally prefer <https://www.manned.org>